doc-content/enterprise-only/vehicle-route-planning/vrp-run-openshift.adoc

```diff
@@ -0,0 +1,52 @@
+[id='vrp-run-openshift_{context}']
+
+= Run OptaWeb Vehicle Routing on {OPENSHIFT} through  Red Hat CodeReady Containers
+
+Linux and macOS users can use the `runOnOpenShift.sh` Bash script to install OptaWeb Vehicle Routing on {OPENSHIFT}. You can use Red Hat CodeReady Containers to set up a single-node OpenShift 4 cluster on your local computer. CodeReady Containers provides a minimal preconfigured OpenShift cluster for development and testing purposes. It is delivered as a Red Hat Enterprise Linux virtual machine.
+
+.Prerequisites
+* OptaWeb Vehicle Routing has been successfully built with Maven as described in xref:download-ref-imp-proc_{context}[].
+* Internet access is available.
+
+
+.Procedure
+. To install and start Red Hat CodeReady Containers, follow the instructions in the link:https://code-ready.github.io/crc/[_Red Hat CodeReady Containers Getting Started Guide_].
+
+. When the OpenShift cluster starts, perform the following steps:
+
+.. Add the OpenShift command-line interface (`oc`) to your `$PATH`:
++
+[source,shell]
+----
+eval $(crc oc-env)
+----
+
+.. Log in as the `developer` user:
++
+[source,shell]
+----
+oc login -u developer -p developer https://api.crc.testing:6443
+----
+
+.. Enter the following command to create a new project where `<PROJECT_NAME>` is the name of your new project:
++
+[source]
+----
+oc new-project <PROJECT_NAME>
+----
+.. If necessary, change directory to the `{PRODUCT_INIT}-{PRODUCT_VERSION_LONG}-optaweb-vehicle-routing/sources` directory.
+
+.. Enter the following command to execute the `runOnOpenShift.sh` script:
++
+[source]
+----
+./runOnOpenShift.sh <OSM_FILE_NAME> <COUNTRY_CODE_LIST> <OSM_FILE_DOWNLOAD_URL>
+----
++
+In this command, replace the following variables:
+
+* `<OSM_FILE_NAME>`: The name of OSM file
```

Comment:
Just a suggestion to add some example how that 
 ./runOnOpenShift.sh my-city.osm.pbf FR https://download.bbbike.org/osm/extract/planet_12.032,53.0171_12.1024,53.0491.osm.pbf

Reply:
I don't have access to that link.

Reply:
yes I could see from the sh script  
./runOnOpenShift.sh belgium-latest.osm.pbf BE http://download.geofabrik.de/europe/belgium-latest.osm.pbf"
- Configures the application to filter geosearch results to Belgium and download the latest Belgium \
OSM extract from Geofabrik.

Reply:
Could you also add a describtion rather that reflects this one
http://github.com/dupliaka/optaweb-vehicle-routing/blob/master/runOnOpenShift.sh#L34-L34
I was a bit confused why should i use name of the same osm file twice in the parameters before i checked that this would be some intermidiate file with that name that script will download.

Reply:
Updated

Reply:
But could you tell why did you change that on 
'./runOnOpenShift.sh belgium-latest.osm.pbf BE' wheras it should be "/runOnOpenShift.sh belgium-latest.osm.pbf BE http://download.geofabrik.de/europe/belgium-latest.osm.pbf"

Reply:
I think I misunderstood. Please check the file now, see if I have it right.

Reply:
Looks fine for me